jax/experimental/callback.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2018 Google LLC
```

Comment:
Copyright 2020? You're living in the past man!

Reply:
Err, I dunno 2020 isn't going so great. I might want to go back!